File: angular-app/src/app/_dtos/chat/UserChatGroupModel.ts
export class UserChatGroupModel {
  listUserName = []
  chatId: number

  update(listUserName: any) {
    this.listUserName = listUserName;
  }

  constructor(listUserName: any[], chatId: number) {
    this.listUserName = listUserName;
    this.chatId = chatId;
  }
}

File: spring-rest-api/src/main/java/com/example/spring/rest/api/model/UserChatGroupModel.java
package com.example.spring.rest.api.model;

import jakarta.validation.Valid;
import jakarta.validation.constraints.NotBlank;
import lombok.Data;

import java.util.List;

@Data
public class UserChatGroupModel {

    @NotBlank(message = "List User Name cannot not blank")
    private List<@Valid String> listUserName;

    @NotBlank(message = "Chat Id cannot not blank")
    private Long chatId;
}
